lua/hop/init.lua

```diff
@@ -119,7 +119,18 @@ local function hint_with(hint_mode, opts)
   })
 
   hint.set_hint_extmarks(hl_ns, hints)
-  keymap.create_jump_keymap(0, opts)
+
+  while h == nil do
+    local key = vim.fn.getchar()
+    if type(key) == 'number' then key = vim.fn.nr2char(key) end
+    if #key == 1 and string.find(opts.keys, key, 1, true) then
+      h = M.refine_hints(0, key)
+    else
+      M.quit(0)
+      vim.api.nvim_feedkeys(key, '', true)
```

Comment:
Can you add a comment stating that we need this as a _key forward_ mechanism?

Reply:
Ya, I will!